### PR TITLE
Fix logging the attachment of exception record to a case

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

Fix logging the attachment of exception record to a case. Also, remove some ambiguity - we had two log entries with the same content, but marking the completion of different things.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java [hunk 1]
```diff
@@ -403,9 +403,9 @@ private void updateSupplementaryEvidence(
         );
 
         log.info(
-            "Successfully attach Exception Record with ID {} to case with id {}",
-            theCase.getId(),
-            exceptionRecordDetails.getId()
+            "Attached Exception Record to a case in CCD. ER ID: {}. Case ID: {}",
+            exceptionRecordDetails.getId(),
+            theCase.getId()
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java]: Wrong argument order was the main reason for this PR

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java [hunk 2]
```diff
@@ -280,7 +280,7 @@ public AttachCaseCallbackService(ServiceConfigProvider serviceConfigProvider,
         }
 
         log.info(
-            "Successfully attached exception record {} to case {}",
+            "Completed the process of attaching exception record to a case. ER ID: {}. Case ID: {}",
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java]: At this point we've done both CCD and payment publishing bit.

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java [hunk 3]
```diff
@@ -403,9 +403,9 @@ private void updateSupplementaryEvidence(
         );
 
         log.info(
-            "Successfully attach Exception Record with ID {} to case with id {}",
-            theCase.getId(),
-            exceptionRecordDetails.getId()
+            "Attached Exception Record to a case in CCD. ER ID: {}. Case ID: {}",
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java]: Main change in the message is making it clear it's just about CCD.

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java (hunk 1 above)
Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java]: I hope some of the reports that I had to do were not based on these log entries 😬

**Pull request merged**